src/Api/WebClient.php

```diff
@@ -1,81 +1,78 @@
 <?php
 
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
 namespace BitBag\SyliusPocztaPolskaShippingExportPlugin\Api;
 
 use BitBag\SyliusShippingExportPlugin\Entity\ShippingGatewayInterface;
 use PocztaPolska\gabarytBiznesowaType;
+use PocztaPolska\getAddresLabelByGuidResponse;
 use PocztaPolska\getEnvelopeBufor;
 use PocztaPolska\getUrzedyNadania;
 use PocztaPolska\pobranieType;
 use PocztaPolska\przesylkaBiznesowaType;
 use PocztaPolska\przesylkaPoleconaZagranicznaType;
 use PocztaPolska\sendEnvelope;
 use PocztaPolska\sposobPobraniaType;
+use SoapFault;
 use Sylius\Component\Core\Model\OrderInterface;
 use Sylius\Component\Core\Model\ShipmentInterface;
 use PocztaPolska\addShipment;
 use PocztaPolska\adresType;
 use PocztaPolska\ElektronicznyNadawca;
 use PocztaPolska\getAddresLabelByGuid;
-use PocztaPolska\paczkaPocztowaType;
 
 final class WebClient implements WebClientInterface
```

Comment:
Can you add changes also for specs?

Reply:
Extracted functions from webclient to factories and generators, webclient should not be tested because its now have only api calls.

Reply:
Done